packages/fx-core/src/plugins/resource/aad/v3/index.ts

```diff
@@ -139,10 +179,8 @@ export class AadAppForTeamsPluginV3 implements v3.FeaturePlugin {
     });
 
     //init aad part in local settings or env state
-    if (!envInfo.state[BuiltInFeaturePluginNames.aad]) {
-      envInfo.state[BuiltInFeaturePluginNames.aad] = {
-        secretFields: ["clientSecret"],
-      };
+    if (!envInfo.state[BuiltInFeaturePluginNames.aad].secretFields) {
```

Comment:
Will `envInfo.state[BuiltInFeaturePluginNames.aad]` be empty?

Reply:
No, solution will guarantee the existence of the corresponding configs